lib/web/device_trust.go

```diff
@@ -44,6 +47,8 @@ func (h *Handler) deviceWebConfirm(w http.ResponseWriter, r *http.Request, _ htt
 	confirmToken := &devicepb.DeviceConfirmationToken{}
 	confirmToken.Id = query.Get("id")
 	confirmToken.Token = query.Get("token")
+	redirectParam := query.Get("redirect_uri")
```

Comment:
nit:

```suggestion
	redirectURI := query.Get("redirect_uri")
```

so it matches the param?